02-express-tutorial/app.js

```diff
@@ -1 +1,107 @@
 console.log('Express Tutorial')
+// Import the Express library
+const express = require('express');
+
+//add the following require statement to the top of the program:
+const { products } = require("./data");
+
+// Creation of the app as returned from calling express()
+const app = express();
+
+//app.use statements for the middleware. You’ll eventually use many kinds of middleware, 
+//but for now the only middleware we are using is express.static().
+app.use(express.static("./public"))
+
+//Add an app.get statement to app.js. It should be after the Express static middleware, but before the “not found” handler. 
+/*app.get('/api/v1/test',(req,res)=>{
+    res.json({ message: "It worked!" });
+})*/
+
+app.get('/api/v1/products',(req,res)=>{
+    res.json(products);
+})
+
+/*So add an app.get statement for the url
+/api/v1/products. Write some code to return JSON for the products
+array.*/
+/*app.get('/api/v1/products/:productID',(req,res)=>{
+    
```

Comment:
Welldone. 
But I noticed you did not handle the stituation where the productID is not found.

Kindly do that

Reply:
Please check, i believe i resolved.

> Welldone. But I noticed you did not handle the stituation where the productID is not found.
> 
> Kindly do that and uncomment the code